src/pages/ReimbursementAccount/EnableStep.js

```diff
@@ -126,16 +108,41 @@ class EnableStep extends React.Component {
                             iconHeight={iconSize}
                             disabled
                             interactive={false}
-                            wrapperStyle={[styles.ph0, styles.mb3]}
+                            wrapperStyle={[styles.cardMenuItem, styles.mv3]}
+                            iconFill={themeColors.success}
                         />
-                        <Text>
+                        <Text style={[styles.mv3]}>
                             {!isUsingExpensifyCard
                                 ? this.props.translate('workspace.bankAccount.accountDescriptionNoCards')
                                 : this.props.translate('workspace.bankAccount.accountDescriptionWithCards')}
                         </Text>
+                        {!isUsingExpensifyCard && (
+                            <Button
+                                text={this.props.translate('workspace.bankAccount.addWorkEmail')}
+                                onPress={() => {
+                                    Link.openOldDotLink(CONST.ADD_SECONDARY_LOGIN_URL);
+                                    User.subscribeToExpensifyCardUpdates();
+                                }}
+                                icon={Expensicons.Mail}
+                                style={[styles.mt4]}
+                                iconStyles={[styles.buttonCTAIcon]}
+                                shouldShowRightIcon
+                                large
+                                success
+                            />
+                        )}
+                        <View style={[styles.mv3]}>
+                            <MenuItem
+                                title={this.props.translate('workspace.bankAccount.disconnectBankAccount')}
+                                icon={Expensicons.Close}
+                                onPress={BankAccounts.requestResetFreePlanBankAccount}
+                                iconFill={themeColors.success}
+                                wrapperStyle={[styles.cardMenuItem]}
+                            />
+                        </View>
```

Comment:
```suggestion
                        <MenuItem
                            title={this.props.translate('workspace.bankAccount.disconnectBankAccount')}
                            icon={Expensicons.Close}
                            onPress={BankAccounts.requestResetFreePlanBankAccount}
                            iconFill={themeColors.success}
                            wrapperStyle={[styles.cardMenuItem, styles.mv3]}
                        />
```

I think the wrapping `View` is not necessary, we can pass the style in `wrapperStyle` directly to the `MenuItem`

Reply:
👍 I think your version is correct @aldo-expensify! This was left over from an earlier version where I was dealing with the heights differently. We should double check the presentation matches the Figma but this should be good.